libs/designsystem/src/lib/components/list/directives/infinite-scroll.directive.ts

```diff
@@ -34,7 +34,14 @@ export class InfiniteScrollDirective implements AfterViewInit, OnDestroy {
   /**
    * If true then {@link scrollEnd} event should NOT be emitted
    */
-  @Input() disabled = false;
+  _disabled: boolean = false;
+  @Input() set disabled(value: boolean) {
+    this.checkScroll$Subscription(value);
```

Comment:
Pretty sure it would suffice just to remove the `disabled` guard at the top of `ngAfterViewInit` as it's already using the `disabled` property as a filter inside the pipe...

Reply:
You're right! - should be resolved by: 96fa29c